File: app/src/main/java/com/webbrowser/JavaScriptInterface.java
package com.webbrowser;

import android.webkit.JavascriptInterface;
import android.webkit.WebView;
import android.widget.Toast;

public class JavaScriptInterface {
    private WebView webView;

    public JavaScriptInterface(WebView webView) {
        this.webView = webView;
    }

    @JavascriptInterface
    public void showToast(String message) {
        Toast.makeText(webView.getContext(), message, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/webbrowser/JavaScriptInterface.kt
package com.webbrowser

import android.webkit.JavascriptInterface
import android.webkit.WebView
import android.widget.Toast

class JavaScriptInterface(private val webView: WebView) {
    @JavascriptInterface
    fun showToast(message: String?) {
        Toast.makeText(webView.context, message, Toast.LENGTH_SHORT).show()
    }

    @JavascriptInterface
    fun printMessage(message: String?) {
        println("Message for Web : ${message}")
    }
}
